src/TodoListItem.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// Add props as a parameter in the TodoListItem function
+// I added ({ todo }) as a parameter in the TodoListItem function
+const TodoListItem = ({ todo }) => {
+    return (
+        // <div>

```

Comment:
You can remove the commented `<div>` tag (and ending tag below). You were correct to directly use `<li>`!